stingray/io.py

```diff
@@ -653,3 +654,39 @@ def get_btis(gtis, start_time=None, stop_time=None):
 def gti_len(gti):
     """Return the total good time from a list of GTIs."""
     return np.sum([g[1] - g[0] for g in gti])
+
+
+def save_object(object, filename):
+    """
+    Pickle a class instance.
+
+    Parameters
+    ----------
+    object: a class instance
+    filename: str (name of the file to be created)
+
+    """
+
+    directory = os.getcwd() + "/data/"
+    if not os.path.exists(directory):
+        os.makedirs(directory)
+
+    filename = directory + filename
+    pickle.dump(object, open(filename, "wb" ) )
+
+def retrieve_object(filename):
+    """
+    Return a pickled class instance.
+
+    Parameters
+    ----------
+    filename: str (name of the file to be retrieved)
+
+    """
+
+    directory = os.getcwd() + "/data/"
+    if not os.path.exists(directory):
+        os.makedirs(directory)
```

Comment:
Looks good to me !

On a side note, It's good to use `amend` and squashing to maintain to good commit history.